programs/axelar-solana-its/src/processor/mod.rs

```diff
@@ -552,14 +567,13 @@ fn process_set_pause_status(accounts: &[AccountInfo<'_>], paused: bool) -> Progr
     Ok(())
 }
 
-fn process_set_trusted_chain(accounts: &[AccountInfo<'_>], chain_name: String) -> ProgramResult {
-    let accounts_iter = &mut accounts.iter();
-    let payer = next_account_info(accounts_iter)?;
-    let program_data_account = next_account_info(accounts_iter)?;
-    let its_root_pda = next_account_info(accounts_iter)?;
-    let system_account = next_account_info(accounts_iter)?;
-
+fn process_set_trusted_chain<'a>(
+    accounts: &'a [AccountInfo<'a>],
+    chain_name: String,
+) -> ProgramResult {
+    let (payer, program_data_account, its_root_pda, system_account) = extract_accounts(accounts)?;
```

Comment:
Perhaps update the name to be consistent with the role management one? Or at least to be a bit more specific since I don't think this function can be used for the other instructions

```suggestion
    let (payer, program_data_account, its_root_pda, system_account) = get_trusted_chain_accounts(accounts)?;
```

Reply:
Done